crates/services/upgradable-executor/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+#![deny(clippy::arithmetic_side_effects)]
+#![deny(clippy::cast_possible_truncation)]
+#![deny(unused_crate_dependencies)]
+#![deny(warnings)]
+
+#[cfg(feature = "wasm-executor")]
+use wasmtime::{
+    Config,
+    Engine,
+    Module,
+};
+
+pub mod config;
+pub mod executor;
+
+#[cfg(feature = "wasm-executor")]
+pub mod instance;
+
+/// The WASM version of the underlying [`fuel_core_executor::executor::ExecutionInstance`].
+#[cfg(feature = "wasm-executor")]
+pub const WASM_BYTECODE: &[u8] = include_bytes!(concat!(
+    env!("OUT_DIR"),
+    "/wasm32-unknown-unknown/release/fuel-core-wasm-executor.wasm"
+));
+
+#[cfg(feature = "wasm-executor")]
+lazy_static::lazy_static! {
```

Comment:
Might not need lazy static anymore.

https://docs.rs/once_cell/latest/once_cell/#safe-initialization-of-global-data

Reply:
Fixed in https://github.com/FuelLabs/fuel-core/pull/1781